airbyte-integrations/connectors/source-postgres/src/test/resources/dummy_config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Why did these files (*.json) need to be added under test/resources in the source-postgres directory? I thought tests needed to be removed from here.